Odata-docs/TOC.yml

```
@@ -390,6 +392,10 @@
             href: /odata/webapiauth/getting-started
           - name: How permissions are applied
             href: /odata/webapiauth/how-permissions-are-applied
+  - name: Performance tips and guidelines
+    items:
+      - name: Marking IEdmModel as immutable
```

```suggestion
      - name: Marking the Edm model as immutable
```
I think saying "the Edm model" is easier on the reader than "IEdmModel"